### PR TITLE
Header

### PR DESCRIPTION
**Pull request by Haidai-Oksana**

### CODE REVIEW

**Comment by Baranov-Oleksandr** [comment]
Everything is well done, thanks for your work.

File: src/styles/components/header.scss [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        width: 110%;
+        bottom: 0;
+        left: -5%;
+        height: 5px;
+        background-color: $color-firm3;
+        transform: translateY(100%);
+        transition: transform $animation-time;
+    }
+    &:hover,
+    &:focus {
```
Comment (AlexandrKolesnikov) [inline src/styles/components/header.scss]: Looks like you forgot active state :) Thank you in advance!

File: src/styles/components/header.scss [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    justify-content: center;
+    align-items: center;
+    padding: 43px 0;
+    overflow: hidden;
+    &:last-child {
+        margin-right: 0;
+    }
+}
+
+.page-header__item-title {
```
Comment (AlexandrKolesnikov) [inline src/styles/components/header.scss]: Could you please rename to `header__menu-link`

File: src/styles/components/header.scss [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    justify-content: center;
+}
+
+.page-header__logo {
+    width: 50px;
+    height: 50px;
+    margin-right: 153px;
+}
+
+.page-header__items {
```
Comment (AlexandrKolesnikov) [inline src/styles/components/header.scss]: Could you please rename to `header__menu`?

File: src/styles/components/header.scss [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    display: flex;
+    flex-wrap: nowrap;
+    justify-content: center;
+}
+
+.page-header__item {
```
Comment (AlexandrKolesnikov) [inline src/styles/components/header.scss]: Could you please rename to `header__menu-item`?

File: src/index.html [hunk 5]
```diff
@@ -12,6 +12,60 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
 </head>
 <body>
+<header class="page-header">
+    <div class="page-header__content">
+        <div class="page-header__logo">
+            <img src="assets/i/s/header-logo.png" alt="Logo">
```
Comment (AlexandrKolesnikov) [inline src/index.html]: Could you please rename **header-logo** -> **logo** cause it maybe used across different places?

**Comment by Baranov-Oleksandr** [comment]
Good job

**Pull request merged**